projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.common.ts

```diff
@@ -23,3 +23,5 @@ export interface IExpansionPanelEventArgs extends IBaseEventArgs {
     event: Event;
     panel: IgxExpansionPanelBase;
 }
+
+export interface IExpansionPanelInteractionEventArgs extends IExpansionPanelEventArgs, CancelableEventArgs {}
```

Comment:
IMO `ExpansionPanelCancelabelEventArgs` is better naming here.
Additionally we can add `collapsing` and `expanding` event in future, where we can use same type for the event arguments.
Also in most places we are calling cancelable event arguments `Something`**Cancelable**`EventArgs`